### ISSUE TITLE
[Steam?] Unreadable Numbers

### ISSUE THREAD

**Issue by Kikithecat12345** [prinzstani/CookieBot#77]
![image](https://user-images.githubusercontent.com/46766794/149637941-92d2f390-4dec-49a7-99e8-72ebf0d26253.png)
The above screenshot shows a completely useless number to the user. Can we make it show something like 1.000 novemdecillion or something? 
EDIT: Forgot to mention: this text shows up when hovering over the version of the bot, and in the notifications at the bottom of the page.

**Comment by prinzstani** [comment]
This screenshot puzzles me. I do not get this message, for me, it looks like
Trying to get achievement: Bake 100 nonillion cookies in one ascension.

I am not sure why you get this huge number. Did you deactivate short numbers? I have tried to deactivate short numbers, but it still shows the short numbers in the notification. Then I restarted the game, and now I get 
Trying to get achievement: Bake 1.00e+32 cookies in one ascension.

So in short, it looks to me that the appearance of this note can be influenced by the normal cookie clicker options, even though I do not know which option you have used to get the result you showed.

**Comment by Kikithecat12345** [comment]
Odd, it could be an issue with the Steam version. I'll update the name of this issue then.

**Comment by pie3636** [comment]
I can confirm having the same issue (on Steam).
![image](https://i.imgur.com/xgz1RRa.png)

Short numbers is enabled in options, and I don't believe I have changed any other setting.

**Comment by prinzstani** [comment]
That is odd. We use the original cookie clicker description of the achievement.
In my version, I have a difference between desc and baseDesc, for example
Game.AchievementsById[279].desc
'Bake <b>100 undecillion</b> cookies in one ascension.'
Game.AchievementsById[279].baseDesc
'Bake <b>100000000000000000000000000000000000000</b> cookies in one ascension.'

The bot uses the desc, which here is the correct one.
Could you check how this unfolds on Steam?

Moreover, what is the description that you see in the note when you finally get the achievement?
What is the description that you see when you hover over the achievement when you finally have it?

**Comment by Kikithecat12345** [comment]
I can check this when I get home. My guess is that it was changed in the Steam version.

**Comment by Kikithecat12345** [comment]
![image](https://user-images.githubusercontent.com/46766794/163079275-e9dbdd0d-98e5-4f75-aa4e-4095754e7e76.png)
@prinzstani

Looking at the object in the console, it seems it was replaced with `.ddesc`:
![image](https://user-images.githubusercontent.com/46766794/163079444-1a5e0088-3293-48fc-99e8-00d36d263813.png)
Looks like code to detect which version you are using and adjusting as such is in order.

**Comment by prinzstani** [comment]
OK, so in Steam .ddesc could work, which is not available in the plain version. So we can use the .ddesc if available and otherwise the .desc.

**Comment by prinzstani** [comment]
ddesc is available in the main version now and is used in the bot (beta for the moment).
